doc/jsk_perception/nodes/alexnet_object_recognition.rst

```diff
@@ -50,9 +50,9 @@ Parameters
   Currently ``alex`` or ``alex_batch_normalization`` is only supported.
   See models in ``$(rospack find jsk_recognition_utils)/python/jsk_recognition_utils/chainermodels``.
 
-* ``~model_file`` (String, Required)
+* ``~model_h5`` (String, Required)
```

Comment:
I think `~model_file` is better, because you can decide deserializers by its extensions.
For exmaple `.npz`.